WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -1148,6 +1152,12 @@ def finalize(self):
     elif res['Value'] and not requestFlag:
       # A request was created while there were none before, change final minor status
       self.__report(minorStatus=JobMinorStatus.PENDING_REQUESTS)
+      requestFlag = True
```

Comment:
I am afraid we shall have the same problem with accounting as we had before, i.e. the final status in accounting will not be `Done`. Remember `sendFailoverRequest()` is sending the accounting!
I don't understand what is wrong with the line 1141 that you removed?

Reply:
I was thinking if we should not remove the sending of the accounting report from `sendFailoverRequest`.

Reply:
We can do that... Maybe the best is for me to merge your PR and see what we end up with, and then move the accounting out of `sendFailoverRequest`?
Still I think the line 1141 was OK and simpler in the logic... 
IIUC you propose to not create a request if sending accounting fails, anyway?

Reply:
I will add to this PR. 

The problem with line 1141 is that we could have had job set "Done" and then followed by "Failed". With the way it's done here, it will not be possible.

The accounting report object ALREADY sets a request if `commit` fails: https://github.com/DIRACGrid/DIRAC/blob/rel-v7r1/AccountingSystem/Client/DataStoreClient.py#L101

Reply:
Oh, I made a mistake and I pushed directly to your branch. So changes are visible in https://github.com/DIRACGrid/DIRAC/pull/4852